src/storybook/src/modul-components/components/file-upload/file-upload.stories.ts

```diff
@@ -1,72 +1,80 @@
+import { storiesOf } from '@storybook/vue';
 import { FILE_UPLOAD_NAME } from '@ulaval/modul-components/dist/components/component-names';
+import { MFile } from '@ulaval/modul-components/dist/utils/file/file';
 import { modulComponentsHierarchyRootSeparator } from '../../../utils';
 
-export default {
-    title: `${modulComponentsHierarchyRootSeparator}${FILE_UPLOAD_NAME}`,
-    parameters: { fileName: __filename }
-};
+storiesOf(`${modulComponentsHierarchyRootSeparator}${FILE_UPLOAD_NAME}`, module)
```

Comment:
@jsroy19  le format a utiliser maintenant dans modul est bel et bien celui des [CSF](https://storybook.js.org/docs/formats/component-story-format/) , je sais que dans Brio vous utilisez plutot la syntaxe avec les `storiesOf`  mais comme elle sera deprecated dans les versions futures de storybook on essait de migrer vers la nouvelle syntaxe.

Reply:
ok, je m'étais basé sur l'Accordion, puisque ca me permettais de setter une function domme props plus facilement que directement dans le template... tu as des suggestion sur comment le faire? (je vais roolbacker les changements pour le reste, pas très compliqué!)

Reply:
En fait ca devrait rien changer au story c'est qu'au lieu de faire 

```
    .add('customValidationFunction', () => ({
```
tu doit faire 
```
export const customValidationFunction =  () => ({
```

Reply:
non je parlais pour passer une prop qui est une function dans le template, j'avais de la difficulté à le faire avec le setup "export const customValidationFunction"
`export const functionProp = (file: MFile) => { return file.name.includes('John') ? Promise.resolve(true) : Promise.resolve(false) };
export const customValidationFunction = () => `
<div>
<h5>Allowed files based on custom validation function</h5>
<span>Allowed filenames that contains the string "John" only - else displays custom message</span>
<m-file-upload custom-validation-message="Le nom du fichier doit contenir les caractères 'John'."
:custom-validation-function="functionProp" ><m-button>File upload</m-button></m-file-upload>
</div>`;`

Reply:
Dans ce cas tu peux definir la fonctions dans les options "data" de la storie.

Tu peux regarder les stories dans form.stories.ts ou on définit l'objet FormGroup dans les  data et on la repasse au template ensuite directement avec  `:form-group="formGroup" `